### ISSUE TITLE
MCT on load should warn about incompatible or untested browsers

### ISSUE THREAD

**Issue by mudinthewater** [nasa/openmct#3191]
Users of Safari in particular keep loading MCT and then reporting it doesn't work, when the base application isn't meant for it. 

Current browsers that work: 

Firefox
Chrome
Edge
Chromium

Everything else should get a warning.

**Comment by charlesh88** [comment]
Agreed. In the case of Safari though, the core issue now is that it _needs_ to work, as it's the foundation of our mobile approach on iOS. It used to work, but isn't right now and many displays are broken (plots, large view, etc.). I'm actively looking into this now.

**Comment by rbiasini** [comment]
Definitely seems like an iOS incompatibility issue, as right now I'm unable to have openMCT loading with any browser on iOS devices (tried Safari, Chrome and Firefox). Is the solution to this issue on the near roadmap?

**Comment by akhenry** [comment]
@rbiasini We need to support mobile, so it will be addressed. Apologies it's not working right now.

It is unfortunately a lower priority for us right now than other issues, but let me get back to you with a timeframe.

FWIW, the reason it's failing in all iOS browsers is because they're all the same rendering engine under the hood (WebKit). Apple don't allow browsers to use their usual rendering engines (Blink, Gecko, etc.) in iOS.
